sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -62,6 +62,9 @@ def test_classification_toy():
         assert np.any(deviance_decrease >= 0.0), \
             "Train deviance does not monotonically decrease."
 
+        leaves = clf.apply(X)
+        assert_equal(leaves.shape, (6, 10, 1))
```

Comment:
If we follow the docstring strictly, we should expect a `(6, 10, 2)` array, since we have 2 classes.